scripts/lint-style.py

```diff
@@ -118,84 +125,72 @@ def skip_string(enumerate_lines):
             # a string literal probably begins and / or ends here,
             # so we skip this line
             if line.count("\"") > 0:
+                yield line_nr, line, *rem, True
                 continue
             if in_string:
+                yield line_nr, line, *rem, True
                 continue
-        yield line_nr, line
-
-def small_alpha_vrachy_check(lines, path):
-    errors = []
-    for line_nr, line in skip_string(skip_comments(enumerate(lines, 1))):
-        if 'ᾰ' in line:
-            errors += [(ERR_SAV, line_nr, path)]
-    return errors
-
-def reserved_notation_check(lines, path):
-    if path.resolve() == RESERVED_NOTATION:
-        return []
-    errors = []
-    for line_nr, line in skip_string(skip_comments(enumerate(lines, 1))):
-        if line.strip().startswith('reserve') or line.strip().startswith('precedence'):
-            errors += [(ERR_RNT, line_nr, path)]
-    return errors
+        yield line_nr, line, *rem, False
 
 def set_option_check(lines, path):
     errors = []
-    for line_nr, line in skip_string(skip_comments(enumerate(lines, 1))):
-        if line.strip().startswith('set_option'):
-            next_two_chars = line.strip().split(' ')[1][:2]
+    newlines = []
+    for line_nr, line, in_comment, in_string in is_in_string(is_in_comment(lines)):
+        if line.strip().startswith('set_option') and not in_comment and not in_string:
+            option_prefix = line.strip().split(' ')[1].split('.')[0]
```

Comment:
```suggestion
            option_prefix = line.strip().split(' ', 2)[1].split('.', 1)[0]
```
to avoid splitting more than needed